recipe/conda_build_config.yaml

```diff
@@ -37,9 +37,9 @@ cross_stdlib_version:
   - 12
 
 clang_version:
-  - 16
-  - 17
-  - 18
+  - 19.1
+  - 18.1
+  - 17.0
```

Comment:
Let's just keep the major version

Reply:
Can we please keep the minor version? It's more accurate (in case something like LLVM 11 ever happens again), and it matches the other "non-native" clang activation feedstocks (clang-win & r_clang).

Or at least, could you flesh out why you think major-only is better?

Reply:
Ping @isuruf

Reply:
Ping @isuruf

Reply:
We've never used a minor version pinning for clang in conda_build_config.yaml right? so I find it unnecessary.

Reply:
Including the minor version is more accurate because while we don't distinguish the patch level (intentionally), a bump in the LLVM minor version _would_ require a separate build. It hasn't happened since LLVM 11.0 -> 11.1, but I think it's better to do so. Tangentially, I find single-component version numbers in conda are a rarity and on top of that, having the minor version would match the other clang (non-default) activations. 

If you don't feel super-strongly about this, could you please humour me on that point?

Reply:
Gentle ping @isuruf

Reply:
Sure